src/hatch_rest_api/riot.py

```diff
@@ -28,13 +28,24 @@ class RestIot(ShadowClientSubscriberMixin):
     blue: int = 0
     white: int = 0
     brightness: int = 0
+    charging_status: int = None
```

Comment:
What is charging status? Is it the battery level? Or an indicator that is it or isn't plugged in?

Reply:
Nm, I see it's use in the other pr, maybe just add a comment here with expected values for others using this library outside ha

Reply:
If you used an enum there wouldn't be a need for comments